### ISSUE TITLE
Error retrieving parent for item

### ISSUE THREAD

**Issue by jossiwolf** [PaoloRotolo/AppIntro#144]
Hello,
I'm trying to use this library, but I've got this two issues:

```
Error retrieving parent for item: No resource found that matches the given name 'android:TextAppearance.Material.Widget.Button.Inverse'.
Error retrieving parent for item: No resource found that matches the given name 'android:Widget.Material.Button.Colored'.
```
<br>
This is my build.gradle:
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 22
    buildToolsVersion "22.0.1"
    defaultConfig {
        applicationId 'x'
        minSdkVersion 15
        targetSdkVersion 22
        versionCode 1
        versionName '0.3'
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
    productFlavors {
    }
}

repositories {
    maven {
        url 'http://dl.bintray.com/amulyakhare/maven'
    }
    mavenCentral()

}

dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    compile 'com.android.support:appcompat-v7:22.2.0'
    compile 'com.android.support:design:22.2.0'
    compile 'com.android.support:recyclerview-v7:22.2.0'
    compile 'com.squareup.okhttp:okhttp:2.5.0'
    compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
    compile 'com.github.afollestad.material-dialogs:core:0.8.5.3@aar'
    compile 'com.balysv:material-ripple:1.0.2'
    compile 'pl.tajchert:waitingdots:0.2.0'
    compile 'com.github.paolorotolo:appintro:3.3.0'
}
```

Anything I'm doing wrong? I'm using AppCompat-v7 22.2.0, Design support and Recyclerview 22.2.0. My Build Tools Version is 22.0.1

Thank you!

**Comment by Andrew-Quebe** [comment]
Ensure you're using the latest version of the build tools and support libraries. I believe it's 23.1.0. If that doesn't solve your issue, come back here. 

Also, where are these popping up? The messages tab?

**Comment by jossiwolf** [comment]
Yes, they were popping up in the messages tab.
Anyhow, I resolved it by using buildTools 23.0.2 and support library 23.1.1 .
This is my new build.gradle, if it helps anyone:

```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 23
    buildToolsVersion '23.0.2'
    defaultConfig {
        applicationId 'x'
        minSdkVersion 15
        targetSdkVersion 23
        versionCode 1
        versionName '0.3'
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
    productFlavors {
    }
}

repositories {
    mavenCentral()
}

dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    compile 'com.android.support:appcompat-v7:23.1.1'
    compile 'com.android.support:design:23.1.1'
    compile 'com.android.support:recyclerview-v7:23.1.1'
    [ Your libraries here]    
    compile 'com.github.paolorotolo:appintro:3.3.0'
}

```
